composer.json

```diff
@@ -66,9 +66,6 @@
             "Tests/"
```

Comment:
Travis is not using an optimized autoloader? Using it could improve performance on the builds?